bundles/org.openhab.binding.pixometer/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pixometer"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="last_reading_date">
+		<item-type>DateTime</item-type>
+		<label>Time of Last Reading</label>
```

Comment:
```suggestion
		<label>Last Reading</label>
```

Reply:
The labels and descriptions don't match with the properties files. Probably better to keep that in sync. For example this label is in the properties file `Reading Date`.